libfwupdplugin/fu-efivar-freebsd.c

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2021 Norbert Kamiński <norbert.kaminski@3mdeb.com>
+ * Copyright (C) 2021 Michał Kopeć <michal.kopec@3mdeb.com>
+ * Copyright (C) 2021 Sergii Dmytruk <sergii.dmytruk@3mdeb.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+#include <efivar.h>
+
+#include "fu-common.h"
+#include "fu-efivar.h"
+
+#include "fwupd-error.h"
+
+/**
+ * fu_efivar_get_path, fu_efivar_get_filename:
+ * Linux-only functions, kept here temporarily for compatibility with
+ * plugins that haven't been adapted yet
+ */
+
```

Comment:
these are static functions, I don't think you need them in the freebsd versions.

Reply:
You're right. Fixed.